terraform/stacks/umccr_data_portal/workflow/main.tf

```diff
@@ -189,6 +189,83 @@ locals {
   }
 }
 
+  transcriptome_wfl_id = {
+    dev  = "wfl.286d4a2e82f048609d5b288a9d2868f6"
+    prod = "wfl.286d4a2e82f048609d5b288a9d2868f6"
+  }
+
+  transcriptome_wfl_version = {
+    dev  = "3.7.5"
+    prod = "3.7.5--99abc32"
+  }
+
+  transcriptome_input = {
+    dev = <<-EOT
+    {
+      "sample-name": null,
```

Comment:
Possible to harmonize using underscore as space separator? Like `sample_name` instead.